wmn-data.json

```diff
@@ -4231,6 +4231,18 @@
         "cat" : "social",
         "valid" : true
        },
+       {
+        "name" : "OpenWRT Forums",
+        "uri_check" : "https://forum.openwrt.org/u/{account}/summary",
+        "post_body" : "",

```

Comment:
Please remove the post_body parameter since this is not a POST

Reply:
Crap. Done.